### PR TITLE
Fix first user flow package snap

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Stepped through each step in the first user flow and fixes the use of the accordion pattern.
- Fixed a number of a11y issues across the pages
- Replace classes with Vanilla ones where possible 

## How to QA
- Go to /first-snap/flutter
- Step through the steps
- Check the steps match [the design](https://zpl.io/2ZDWkAL)
- Try the accordions work as expected

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/3925

## Screenshots
image

### CODE REVIEW

File: static/js/public/accordion.js [hunk 1]
```diff
@@ -9,9 +9,10 @@
   @param {Boolean} show Whether to show or hide the accordion panel.
 */
 export const toggleAccordion = (element, show) => {
+  console.log("toggleAccordion", element, show);
```
Comment (Lukewh) [inline static/js/public/accordion.js:12]: oops

File: static/js/public/accordion.js [hunk 2]
```diff
@@ -9,9 +9,10 @@
...
   element.setAttribute("aria-expanded", show);
   document
-    .querySelector(element.getAttribute("aria-controls"))
+    .querySelector(`#${element.getAttribute("aria-controls")}`)
```
Comment (Lukewh) [inline static/js/public/accordion.js:15]: could just use `document.getElementById(element.getAttribute("aria-controls"))` ?

**Review by Lukewh — CHANGES_REQUESTED** [review]
1 console log to remove and 1 suggestion!

File: static/js/public/accordion.js (hunk 2 above)
Comment (steverydz) [reply static/js/public/accordion.js:15]: I'd say that using `querySelector` is preferable. I think we should standardise on that as a code style thing.

Comment (Lukewh) [reply static/js/public/accordion.js:14]: FINE

**Review by Lukewh — APPROVED** [review]
:+1: LGTM

**Comment by AnnSoMuller** [comment]
On the look and feel, It should follow the Vanilla pattern https://vanillaframework.io/docs/examples/patterns/accordion/headings for accordeon and Have the "-" and"+" aligned with text :)

**Comment by AnnSoMuller** [comment]
there is also a misalignment within the steps ;)

**Comment by anthonydillon** [comment]
@AnnSoMuller I have aligned the icons and headers in the accordions as Vanilla shows it. 

The misalignment of the content is by design from Vanilla:
image

**Comment by AnnSoMuller** [comment]
Good catch @anthonydillon ^^
Good to go then (for me) +1

**Comment by anthonydillon** [comment]
@Lukewh @jkfran are you ok to do the QA?

**Pull request merged**